test/e2e/framework/skipper/skipper.go

```diff
@@ -136,28 +127,15 @@ func SkipUnlessAtLeast(value int, minValue int, message string) {
 	}
 }
 
-// SkipUnlessLocalEphemeralStorageEnabled skips if the LocalStorageCapacityIsolation is not enabled.
-func SkipUnlessLocalEphemeralStorageEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(localStorageCapacityIsolation) {
-		skipInternalf(1, "Only supported when %v feature is enabled", localStorageCapacityIsolation)
-	}
-}
-
-func SkipUnlessDownwardAPIHugePagesEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(downwardAPIHugePages) {
-		skipInternalf(1, "Only supported when %v feature is enabled", downwardAPIHugePages)
-	}
-}
-
-func SkipUnlessExecProbeTimeoutEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(execProbeTimeout) {
-		skipInternalf(1, "Only supported when %v feature is enabled", execProbeTimeout)
+func SkipUnlessFeatureGate(gate featuregate.Feature) {
```

Comment:
Following Go's best practices, please add a short comment documenting what this function does. Follow the syntax:
```go
// MyFunction does something if X is enabled.
```